sdk/tests/client/node_api.rs

```diff
@@ -245,16 +245,7 @@ async fn test_get_milestone_by_id() {
     let node_info = client.get_info().await.unwrap();
 
     let r = client
-        .get_milestone_by_id(
-            &node_info
-                .node_info
-                .status
-                .latest_milestone
-                .milestone_id
-                .unwrap()
-                .parse()
-                .unwrap(),
-        )
+        .get_milestone_by_id(&node_info.node_info.status.latest_milestone.milestone_id.unwrap())
```

Comment:
Maybe use this opportunity to document the unwrap? why is this an `Option` again? How can a `latest_milestone` instance not have an ID?

Reply:
Honestly, these are tests, there are thousands of undocumented unwraps in tests, it's not worth it...